src/main/kotlin/no/skatteetaten/aurora/boober/service/RedeployContext.kt

```diff
@@ -0,0 +1,68 @@
+package no.skatteetaten.aurora.boober.service
+
+import com.fasterxml.jackson.databind.node.ArrayNode
+import no.skatteetaten.aurora.boober.service.openshift.OpenShiftResponse
+import no.skatteetaten.aurora.boober.utils.openshiftKind
+import no.skatteetaten.aurora.boober.utils.openshiftName
+
```

Comment:
In this class you might want to look at the changes i have done in my 1980 PR. I introduce kubernetes-model there. Then you could marhsal IS and DC as Domain objects. The code here would be a lot cleaner IMHO

Reply:
This class does not really need a list of OpenShift Responses. It needs an optional DC and an optional IS.

Reply:
It is very confusing to me with english / norwegian in the same PR :)

I have moved out the code to find IS and DC, so RedeployContext takes these as input to the constructor.

In ragards to the kubernetes-dsl, I totally agree that this probably should be updated to use it. However, this task was created to improve the testability of this class. I am not a fan of taking too big steps at once, it only ends up with long lived branches. I would rather like to get this task done and then create a new task to update it to use the kubernetes-dsl. Do you think this seems like a good approach?